specs/windows/windows_search.table

```diff
@@ -0,0 +1,18 @@
+table_name("windows_search")
+description("Run searches against the windows system index database.")
```

Comment:
Can we link to the microsoft query docs? (or is it a horrific URL)

Reply:
done